test/pom.xml

```diff
@@ -134,13 +134,6 @@
 			<artifactId>junit-jupiter</artifactId>
 			<scope>test</scope>
 		</dependency>
-		<dependency>
-			<groupId>org.frankframework</groupId>
-			<artifactId>frankframework-core</artifactId>
-			<version>${project.version}</version>
-			<type>test-jar</type>
-			<scope>test</scope>
-		</dependency>
```

Comment:
I thought I was doing a 'good thing' by removing unused stuff from the classpath. But it turns out that the IafTestInitalizer tests were depending on these test-classpath resources.